src/renderer/containers/BeaconNodes/BeaconNodeButtons.tsx

```diff
@@ -95,20 +119,49 @@ export const BeaconNodeButtons: React.FunctionComponent<IBeaconNodeButtonsProps>
             <div className='row buttons'>
                 {image ? (
                     isRunning ? (
-                        <ButtonInverted onClick={(): Promise<void> => onStopClick(image)}>Stop</ButtonInverted>
+                        <>
+                            <ButtonInverted onClick={(): void => setConfirmModal(Modal.stop)}>Stop</ButtonInverted>
+                            <ButtonDestructive onClick={(): void => setConfirmModal(Modal.kill)}>
+                                Kill
+                            </ButtonDestructive>
+                        </>
                     ) : (
-                        <ButtonPrimary onClick={(): Promise<void> => onStartClick(image)}>Start</ButtonPrimary>
+                        <ButtonPrimary onClick={(): void => setConfirmModal(Modal.start)}>Start</ButtonPrimary>
                     )
                 ) : null}
-                <ButtonDestructive onClick={onRemoveClick}>Remove</ButtonDestructive>
+                <ButtonDestructive onClick={(): void => setConfirmModal(Modal.remove)}>Remove</ButtonDestructive>
             </div>
 
             <ConfirmModal
-                showModal={confirmModal}
+                showModal={confirmModal === Modal.remove}
                 question={"Are you sure?"}
                 description={"This will remove all your beacon chain data."}
```

Comment:
I just noticed this, we should maybe store data on separate volume and maybe add button to delete chain data.
Otherwise we won't be able to update beacon node without resyncing. 

This is quite annoying part of managing docker node.

cc @morrigan

Reply:
Ok, let's discuss this on a call and create a separate issue for that as it's not the scope of this PR.